openrpc.json

```
@@ -646,9 +648,17 @@
       }
     },
     "tags": {
-      "Metamask": {
-        "name": "Metamask",
-        "description": "Metamask specific methods"
+      "MetaMask": {
+        "name": "MetaMask",
+        "description": "MetaMask-specific methods"
+      },
+      "Restricted": {
+        "name": "Restricted",
+        "description": "Restricted methods"
```

We could add a bit more to the description, maybe this (from the docs):

```suggestion
        "description": "Restricted methods. If a method is restricted, the caller must have the corresponding permission via `wallet_requestPermissions` in order to call it."
```